components/selection/selection-summary.js

```diff
@@ -0,0 +1,42 @@
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { bodyCompactStyles } from '../typography/styles.js';
+import { LocalizeCoreElement } from '../../lang/localize-core-element.js';
+import { selectionStates } from './selection-mixin.js';
+import { SelectionSubscriberMixin } from './selection-subscriber-mixin.js';
+
+/**
+ * A summary showing the current selected count.
+ */
+class Summary extends LocalizeCoreElement(SelectionSubscriberMixin(LitElement)) {
+
+	static get properties() {
+		return {
+			/**
+			 * Text to display if no items are selected.
+			 */
+			noSelectionText: { type: String, attribute: 'no-selection-text' }
```

Comment:
Should we provide a nice default for this?

Reply:
I think there is a bug in the rendering actually - it should default to the existing message `0 selected` or the localized likes thereof.  The `noSelectionText` is just to override that, which `d2l-list-header` does since is places the summary next to the select-all and says "Select All".